anomalib/utils/cv/connected_components.py

```diff
@@ -0,0 +1,51 @@
+"""Connected component labeling."""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import cv2
+import numpy as np
+import torch
+from kornia.contrib import connected_components
+from torch import Tensor
+
+
+def connected_components_gpu(binary_input: Tensor, num_iterations: int = 1000) -> Tensor:
```

Comment:
How about naming `binary_input` as `image`? This is how `kornia` names this. Also `connected_components_cpu` function too